tox.ini

```diff
@@ -4,6 +4,7 @@ envlist=py27,py37,docs,linting
 [testenv]
 deps=
     setuptools_scm
+    py
```

Comment:
This is needed as some tests still use `py` (for example `py.io`). Didn't dive too deep to see if it was easy to replace by something else, but seems easier to wait for someone to drop Python 2.7/<3.6 to deal with that.